updating/installing-update-service.adoc

```diff
@@ -51,6 +51,12 @@ include::modules/update-service-create-service-web-console.adoc[leveloffset=+2]
 
 include::modules/update-service-create-service-cli.adoc[leveloffset=+2]
 
+[NOTE]
+====
+For the policy engine route, if you see `ReconcileCompleted` status as `false`  with the reason `CreateRouteFailed` caused by `host must conform to DNS 1123 naming convention
+and must be no more than 63 characters`, try creating the Update Service with a shorter name.
+====
```

Comment:
I'd suggest to tell users that the route name is formed by <update-service-name>-policy-engine-route-<update-service-namespace> so that users can assess what length of name can be used.
Besides, as the namespace is not hardcoded to `openshift-update-service`, if users are using a long enough namespace, such as equal to 63 characters, we need to ask user to shorten namespace as well.

Reply:
There is an existing bug when the name space is not openshift-update-service. We are working on fixing that bug. 

I do not think we should go to the specifics around how to reduce the route name as there are several ways users can do it. Also it is not hard to figure out ways to reduce the route name. As we should just inform about the limitation to keep them aware so that they are not surprised by it.